openedx/core/djangoapps/credit/models.py

```diff
@@ -709,3 +708,28 @@ def __unicode__(self):
             provider=self.provider.provider_id,
             status=self.status,
         )
+
+
+class CreditApiConfig(ConfigurationModel):
```

Comment:
Make this more general (`CreditConfig`) and update the docstring to indicate that this is meant to manage credit configuration.